--- a/manjaro-zsh-config
+++ b/manjaro-zsh-config
@@ -23,8 +23,8 @@ zstyle ':completion:*' cache-path ~/.zsh/cache
 HISTFILE=~/.zhistory
 HISTSIZE=10000
 SAVEHIST=10000
-export EDITOR=/usr/local/bin/nano
-export VISUAL=/usr/local/bin/nano
+export EDITOR=/usr/pkg/bin/nano
+export VISUAL=/usr/pkg/bin/nano
 WORDCHARS=${WORDCHARS//\/[&.;]}                                 # Don't consider certain characters part of the word
 
 
@@ -60,7 +60,7 @@ alias cp="cp -i"                                                # Confirm before
 alias df='df -h'                                                # Human-readable sizes
 alias free='top -d1 | head -5 | tail -2'                        # Show sizes in MB
 alias gitu='git add . && git commit && git push'
-alias ls='ls -G'                                                # Colorized output
+#alias ls='ls -G'                                                # Colorized output
 
 # Theming section  
 autoload -U compinit colors zcalc
@@ -80,9 +80,9 @@ export LESS=-R
 
 ## Plugins section: Enable fish style features
 # Use syntax highlighting
-source /usr/local/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
+source /usr/pkg/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
 # Use history substring search
-source $HOME/.config/zsh.d/zsh-history-substring-search.zsh
+source /usr/pkg/share/zsh-history-substring-search/zsh-history-substring-search.zsh
 # bind UP and DOWN arrow keys to history substring search
 zmodload zsh/terminfo
 bindkey "$terminfo[kcuu1]" history-substring-search-up
