commit 00b5ec3b5c23096119daef22a91e269e0b0cb12e
Author: kikadf <kikadf.01@gmail.com>
Date:   Thu Jul 30 16:39:58 2026 +0200

    Disable gitstatusd

diff --git a/p10k-portable.zsh b/p10k-portable.zsh
index 5eea0be..ba7b79f 100644
--- a/p10k-portable.zsh
+++ b/p10k-portable.zsh
@@ -32,7 +32,7 @@
   typeset -g POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(
     # os_icon               # os identifier
     dir                     # current directory
-    vcs                     # git status
+    # vcs                   # git status
     prompt_char             # prompt symbol
   )
 
@@ -1583,7 +1583,7 @@
   #   - always:   Trim down prompt when accepting a command line.
   #   - same-dir: Trim down prompt when accepting a command line unless this is the first command
   #               typed after changing current working directory.
-  typeset -g POWERLEVEL9K_TRANSIENT_PROMPT=off
+  typeset -g POWERLEVEL9K_TRANSIENT_PROMPT=always
 
   # Instant prompt mode.
   #
diff --git a/p10k.zsh b/p10k.zsh
index 6b1eb07..e299291 100644
--- a/p10k.zsh
+++ b/p10k.zsh
@@ -33,7 +33,7 @@
   typeset -g POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(
     os_icon                 # os identifier
     dir                     # current directory
-    vcs                     # git status
+    # vcs                   # git status
     # prompt_char           # prompt symbol
   )
 
@@ -1695,7 +1695,7 @@
   #   - always:   Trim down prompt when accepting a command line.
   #   - same-dir: Trim down prompt when accepting a command line unless this is the first command
   #               typed after changing current working directory.
-  typeset -g POWERLEVEL9K_TRANSIENT_PROMPT=off
+  typeset -g POWERLEVEL9K_TRANSIENT_PROMPT=always
 
   # Instant prompt mode.
   #
